openedx/adg/lms/applications/models.py

```diff
@@ -193,6 +193,20 @@ def is_user_business_line_admin(cls, user):
         """
         return cls.objects.filter(group__user=user).exists()
 
+    @classmethod
+    def get_business_lines_for_user(cls, given_user):
```

Comment:
Do we need to name the parameter `given_user`? Even if we name it `user`, it's obvious that it'd be a given user, since its being received in the parameters.